diagnostic.md

```diff
@@ -45,7 +45,7 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+3 times
```

Comment:
Correct on having it run 3 times, but you missed what the value of x is. The value of x after 3 loops would be 8.